packages/create-amplify/src/logger.ts

```diff
@@ -1,18 +1,29 @@
 import yargs from 'yargs';
+import * as os from 'os';
 
 /**
  * A logger that logs messages to the console.
  */
 export class Logger {
   /**
-   * Creates a new Logger instance.
+   * Properties for ellipsis animation
+   */
+  private timer: ReturnType<typeof setTimeout>;
+  private refreshRate: number;
+  private animateMessage: string;
+
+  /**
+   * Creates a new Logger instance. Injecting stdout for testing.
    * @param console The console to log to.
    * @param minimumLogLevel The minimum log level to log.
    */
   constructor(
     private readonly console: Console = global.console,
-    private readonly minimumLogLevel: LogLevel = LogLevel.INFO
-  ) {}
+    private readonly minimumLogLevel: LogLevel = LogLevel.INFO,
+    private readonly stdout = process.stdout
+  ) {
+    this.refreshRate = 500; // every 0.5 seconds
+  }
```

Comment:
Update the jsdoc

Reply:
Done in [fb89cac](https://github.com/aws-amplify/amplify-backend/pull/595/commits/fb89cacbc4748373aa55231303d37aa8d05dccbc)